src/table.ts

```diff
@@ -1685,6 +1539,21 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
       typeof optionsOrCallback === 'function' ? optionsOrCallback : cb!;
     const gaxOptions =
       typeof optionsOrCallback === 'object' ? optionsOrCallback : {};
+    if (
+      gaxOptions?.retry === undefined &&
+      gaxOptions?.retryRequestOptions === undefined
+    ) {
+      // For now gax will not do any retries for table.sampleRowKeys unless
+      // the user specifically provides retry or retryRequestOptions in the
+      // call.
+      // Moving retries to gax for table.sampleRowKeys will be done in a
```

Comment:
same question as for mutate rows

Reply:
In sampleRowKeys calls, there is currently no custom retry logic which is a different situation than `table.mutate`. So in this case I would be ok with removing this if block and just counting on the default behaviour in gax for now since we don't have retry logic to fall back on. Should we remove this block for now?

Reply:
Given that there's no custom retry logic, I think it's fine - worst case scenario, we get a report of funky behavior, and we can add the block back in.